lib/seattleflu/db/cli/command/etl/clinical.py

```diff
@@ -209,24 +209,20 @@ def encounter_details(document: dict) -> dict:
             },
         }
 
-def age(age: float) -> dict:
+def age(age: float) -> str:
     """
-    Given an *age*, return a dict containing its 'value' and a boolean for
-    'ninetyOrAbove'.
+    Given an *age*, return a string to fit the interval format.
 
-    Currently applys math.ceil() to age to match the age from Audere.
-    This may change in the future as we push to report age in months for
-    participants less than 1 year old.
-
-    If no value is given for *age*, then will just retun None.
+    If no value is given for *age*, then will just return None.
     """
     if age is None:
         return None
+    if age > 90:
+        return "90 years"
+    year = int(age)
+    month = round((age % 1) * 12)
+    return f"{year} years {month} months"
 
-    return {
-        "value": min(ceil(age), 90),
-        "ninetyOrAbove": ceil(age) >= 90
-    }
 
 def race(race_name: str) -> list:
```

Comment:
I can't comment on a line below which is out of context, but when I try running the new clinical ETL on the production data, I run into the `UnknownRaceError`. Were you able to successfully run this on your computer, or do you find the same error?

Reply:
Re: the UnknownRaceError. That made me think the production data have the race 'Unknown' instead of NULL (as newly defined in `preprocess.py` using pandas `na_values` on read).

Reply:
Yep, I believe it does because we loaded the data before that change to preprocess was made.